gitpod-io/gitpod
# [installer] add snapshot class into PVC config for ws-manager

## Description

add snapshot class into PVC config for ws-manager

## Related Issue(s)

Part of https://github.com/gitpod-io/gitpod/pull/9475

## How to test

Run installer and verify that it added default snapshot class name ("") to ws-manager configmap

## Release Notes

```release-note
NONE
```

## Documentation
